packages/wrangler/src/type-generation.ts

```diff
@@ -144,7 +144,7 @@ export function constructType(
 		if (useRawVal) {
 			return `${typeKey}: ${value};`;
 		}
-		return `${typeKey}: "${value.replace(/"/g, '\\"')}";`;
+		return `${typeKey}: "${JSON.stringify(value)}";`;
```

Comment:
I don't think this needs wrapped in quotation marks?
```suggestion
		return `${typeKey}: ${JSON.stringify(value)};`;
```